test/models/observation_test.rb

```diff
@@ -88,6 +87,37 @@ def test_name_been_proposed
     assert(!observations(:coprinus_comatus_obs).name_been_proposed?(names(:conocybe_filaris)))
   end
 
+  def test_vote_counts
+    obs = observations(:owner_only_favorite_eq_consensus)
+    assert_equal(0, obs.users_votes(users(:rolf)).count)
```

Comment:
It's not clear to me why this test cares that Rolf hasn't voted on this observation.  I'd rather see obs.user (than users(:dick)) compared to obs.owner_favorite_votes.count.